index.php

```diff
@@ -12,7 +17,32 @@
             <h1> Flock Information Database Home </h1>
         </div>        
         <div class="content">
-            <h2> Stick a login form here </h2>
+            
+            <center>
+                <h3>User Login</h3>
+                <hr />
+                <form method="post" action="redirectPostLogin.php">
+                    <table border="0">
+                        <tr>
+                            <td><b>Username</b></td>
+                            <td><input type="text" name="username">
+
+                        </tr>
+                        <tr>
+                            <td><b>Password</b></td>
+                            <td><input name="password" type="password"></input></td>
+                        </tr>
+                        <br />
+                        <tr>
+                            <td><input type="submit" value="Submit" />
+
+                            <td><input type="button" value="Register" onclick="gotoregister()">
```

Comment:
This button probably belongs outside the form tag since it doesn't contribute to it, it's a separate action.